shell/edit/workload/__tests__/Upgrading.test.ts

```diff
@@ -22,7 +22,7 @@ describe('component: Upgrading', () => {
     input.setValue(newValue);
     input.trigger('blur');
 
-    expect(wrapper.props('value')[key]).toBe(newValue);
+    expect(wrapper.props('value')?.[key]).toBe(newValue);
```

Comment:
Was this intentional? Should we know if the value is present in tests?

I might be missing something there.

Reply:
It was just to do not break the test while doing the migration and get non-formal errors.
It will fail anyway.